rest/route/service_test.go

```diff
@@ -1017,20 +1018,27 @@ func TestTaskResetPrepare(t *testing.T) {
 	Convey("With handler and a project context and user", t, func() {
 		trh := &taskRestartHandler{}
 
+		testTask := task.Task{
+			Id:           "testTaskId",
+			Activated:    false,
+			Secret:       "initialSecret",
+			DispatchTime: time.Now(),
+			BuildId:      "b0",
+			Version:      "v1",
+			Status:       evergreen.TaskSucceeded,
+			Priority:     0,
+		}
+
 		projCtx := serviceModel.Context{
-			Task: &task.Task{
-				Id:        "testTaskId",
-				Priority:  0,
-				Activated: false,
-			},
+			Task: &testTask,
 		}
 		u := user.DBUser{
 			Id: "testUser",
 		}
 		ctx := context.Background()
 
 		Convey("should error on empty project", func() {
-			req, err := http.NewRequest(http.MethodPost, "task/testTaskId/restart", &bytes.Buffer{})
+			req, err := http.NewRequest(http.MethodPatch, "task/testTaskId/restart", &bytes.Buffer{})
```

Comment:
Why did you change these to patch? It looks like the restart route _is_ POST.

Reply:
I was testing out different things that were causing an issue, I forgot to change them back! Going to revert that change